x-pack/plugins/cases/public/components/create/form_context.test.tsx

```diff
@@ -285,6 +286,18 @@ describe('Create case', () => {
       );
     });
 
+    it('should select LOW as the default severity', async () => {
+      const renderResult = mockedContext.render(
+        <FormContext onSuccess={onFormSubmitSuccess}>
+          <CreateCaseFormFields {...defaultCreateCaseForm} />
+          <SubmitCaseButton />
+        </FormContext>
+      );
+      expect(renderResult.getByTestId('caseSeverity')).toBeTruthy();
+      // there should be 2 low elements. one for the options popover and one for the displayed one.
+      expect(renderResult.getAllByTestId('case-severity-selection-low').length).toBe(2);
```

Comment:
Can you also submit and check that the `postCase` has been called with `severity: low`

Reply:
Good one.

Reply:
Actually. that case is already tested in the previous tests about submit (line 192) and all the submit tests with connectors.

Reply:
Correct! I missed that. Thanks.